eslint/cactbot-output-strings.js

```diff
@@ -19,10 +19,13 @@ const ruleModule = {
     messages: {
       noOutputStrings: 'no outputStrings in trigger',
       notFoundProperty: 'no \'{{prop}}\' in \'{{outputParam}}\'',
+      notFoundTemplate: '`output.{{outputParam}}(data)` doesn\'t take property \'{{prop}}\' on `data`',
```

Comment:
```suggestion
      notFoundTemplate: '`output.{{prop}}(...)` doesn\'t have template \'{{template}}\'.',
```

This was confusing to read compared to `notFoundProperty`, because `outputParam` on `notFoundProperty` refers to the `output` variable, but `outputParam` on `notFoundTemplate` refers to the property on `output`.  I think this naming makes things more clear.  What do you think?